src/main/resources/resources/logisim/strings/soc/soc_zh.properties

```diff
@@ -131,16 +131,16 @@ ElfSymTableReadError = 无法读取符号表。
 #
 # file/ProcessorReadElf.java
 #
-ProcReadElf64BitNotSupportedYet = Logisim尚不支持64位ELF文件。
-ProcReadElfArchError = ELF文件用于\“%s\”处理器，而它应该用于\“%s\”处理器。
-ProcReadElfEndianMismatch = 字节顺序不匹配，ELF文件是\“%s\”，而处理器请求的是\“%s\”。
+ProcReadElf64BitNotSupportedYet = Logisim 尚不支持 64 位 ELF 文件。
+ProcReadElfArchError = ELF 文件用于\“%s\”处理器，而它应该用于\“%s\”处理器。
+ProcReadElfEndianMismatch = 字节顺序不匹配，ELF 文件是\“%s\”，而处理器请求的是\“%s\”。
 ProcReadElfErrorOpeningFile = 无法打开精灵文件
-ProcReadElfLoadableSectionNotFound = 在ELF文件中找不到可加载节。
+ProcReadElfLoadableSectionNotFound = 在 ELF 文件中找不到可加载节。
 ProcReadElfLoadableSectionReadError = 读取可加载节时出错。
 ProcReadElfLoadableSectionSizeError = 可加载段中的大小不匹配。
-ProcReadElfLoadableSectionTooBig = Logisim支持的大小越大，可加载段的大小越大。
-ProcReadElfMemoryError = 将可加载节写入内存时出错。\n请确保您有可用于内存区域的SoC内存组件：\n%s.%s
-ProcReadElfNotExecutable = ELF文件不是可执行文件
+ProcReadElfLoadableSectionTooBig = Logisim 支持的大小越大，可加载段的大小越大。
+ProcReadElfMemoryError = 将可加载节写入内存时出错。\n 请确保您有可用于内存区域的 SoC 内存组件：\n%s.%s
```

Comment:
Remove space before and after the two occurences of `\n`.

The two `%s` get replaced by memory addresses, does `%s.%s` correctly signify a memory region? In the English localisation, `%s...%s` is used.

Reply:
Another DeepL issue. It translated nearly all "..."s to ".". Fixed.